docs/extensibility/resources.md

```diff
@@ -176,6 +184,59 @@ renders the same set of data as:
 ]
 ```
 
+## relations section
+
+This section contains an object that maps a relation name to relation configuration object. Relation name preceded by dollar sign '\$' can be then used in `path` expression.
```

Comment:
```suggestion
The `relations` section contains an object that maps a relation name to a relation configuration object. The relation name preceded by a dollar sign '\$' can be then used in the `path` expression.
```

"Can" it be used, or "is" it used? Does "can" imply it's optional or there are other alternatives? Maybe we can drop the "can".

Reply:
Alright! Thank you for clarification :)